CONTRIBUTING.md

```diff
@@ -24,10 +24,9 @@ Then, [tell folks what you'll be working on](https://gitter.im/okTurtles/group-i
 Everyone must follow the rules below (inspired by the [C4.1](http://hintjens.com/blog:93) process) to submit _code_ (documentation may be edited directly by maintainers):
 
 1. **Always** work in your own fork and submit pull requests (PRs) to `master`.
-2. **Always** submit a _minimal and accurate answer_ to any issue. The simplest solution is the best solution.
```

Comment:
I removed this line as it's repeated with a SHOULD in the next section, so they had conflicting requirement levels

Reply:
ACK :+1: